.github/workflows/ci.yml

```diff
@@ -5,21 +5,21 @@ jobs:
     name: Lighthouse
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Lighthouse
-        uses: shopify/lighthouse-ci-action@v1
+        uses: shopify/lighthouse-ci-action@v1.1.1
         with:
           store: ${{ secrets.SHOP_STORE_OS2 }}
           password: ${{ secrets.SHOP_PASSWORD_OS2 }}
           access_token: ${{ secrets.SHOP_ACCESS_TOKEN }}
-          product_handle: puppy
```

Comment:
By default `lighthouse-ci-action` will use the first product. `puppy` only works for the `os2-demo` shop, but I'm planning to switch to `theme-design-ci`.